emctl/cmd/client/command/meshinstall/controlpanel/deploy.go

```diff
@@ -153,7 +153,7 @@ func checkEasegressControlPlaneStatus(ctx *installbase.StageContext) error {
 	// Wait a fix time for the Easegress cluster to start
 	time.Sleep(time.Second * 10)
 
-	entrypoints, err := installbase.GetMeshControlPanelEntryPoints(ctx.Client, ctx.Flags.MeshNamespace,
+	entrypoints, err := installbase.GetMeshControlPanelEndpoints(ctx.Client, ctx.Flags.MeshNamespace,
```

Comment:
It's my fault, ControlPanel is a typo, pls help me to align it to ControlPlane.   I wrongly use the name of `panel` in someplace, please fix it also.

Reply:
Fixed